colosseum/constants.py

```diff
@@ -45,6 +48,20 @@ def __str__(self):
         return ", ".join(sorted(choices))
 
 
+class OtherProperty:
+    """A class to refer to a callable to perform operations with other properties."""
```

Comment:
@freakboy3742 we could generalize this a little bit so that if we initialized with a property name we use that, otherwise the `value` needs to be overriden. 

Like what was done in https://github.com/beeware/colosseum/pull/55

If you agree with the approach I can update this to cover both cases and close PR55